_docs_v7/Physical-Definition.md

```diff
@@ -79,7 +82,7 @@ For all schemes, as reference values for the density and temperature the free-st
 
 ## Flow Condition (Incompressible) ##
 
-| Solver | Version | 
+| Solver | Version |
 | --- | --- |
 | `INC_EULER`, `INC_NAVIER_STOKES`, `INC_RANS` | 7.0.0 |
```

Comment:
Do these 7.0.0 need to be updated?

Reply:
The info there is still valid for v7, what is missing here that I noticed are the NEMO solvers